docs/iidm/model/line.md

```diff
@@ -7,20 +7,55 @@ The `com.powsybl.iidm.network.Line` interface is used to model an AC line. A lin
 
 # Characteristics
 
-| Attribute | Type | Required | Default value | Description |
-| --------- | ---- | -------- | ------------- | ----------- |
-| R | double | yes | - | The series resistance |
-| X | double | yes | - | The series reactance |
-| G1 | double | yes | - | The first side shunt conductance |
-| B1 | double | yes | - | The first side shunt susceptance |
-| G2 | double | yes | - | The second side shunt conductance |
-| B2 | double | yes | - | The second side shunt susceptance |
+| Attribute | Type | Unit | Required | Default value | Description |
+| --------- | ---- | ---- | -------- | ------------- | ----------- |
+| id | string | n/a | yes | - | Unique identifier of the line|
+| name | string | n/a | no | - | Humanly readable name of the line|
+| R | double | yes | $$\Omega\$$ | - | The series resistance |
```

Comment:
in the line model image, R and X are not capitalized (nor b and g), consequently you should use r and x